### PR TITLE
Yyx/327 test sherlock

### PR DESCRIPTION
**Pull request by yixiangyin**
# Pull Request Details

## What issue does this PR address

- [A one line description of this PR]

- Link to associated issue: #

## Checklist

Complete the check-list below to ensure your branch is ready for PR.

Flutter Style Guide: https://survivor.togaware.com/gnulinux/flutter-style.html

- [ ] Screenshots included in linked issue
- [ ] Changes adhere to the style and coding guideline
- [ ] No confidential information
- [ ] No duplicated content
- [ ] No lint check errors related to your changes (`make prep` or `flutter analyze lib`)
- [ ] Pre-exisiting lint errors noted: [HERE]
- [ ] No integrated test errors
- [ ] Tested on at least one device
  - [ ] Android Phone
  - [ ] Android Emulator
  - [ ] Chrome on Android
  - [ ] Chrome
  - [ ] iOS
  - [ ] Linux
  - [ ] MacOS
  - [ ] Windows
- [ ] Added two reviewers

## Finalising

Once PR discussion is complete and reviewers have approved:

- [ ] Merge dev into the branch
- [ ] Resolve any conflicts
- [ ] Add one line summary into CHANGELOG.md
- [ ] Bump appropriate version number in pubspec.yaml
- [ ] Push to git repository and review
- [ ] Merge PR into dev (gjwgit)

### CODE REVIEW

File: integration_test/utils/open_large_dataset.dart
Comment (gjwgit) [inline integration_test/utils/open_large_dataset.dart:1]: I prefer one file per function.

**Review by gjwgit — CHANGES_REQUESTED** [review]
Basic load datasaet test. Thanks. 

Please split openDatsetByPath to open_dataset_by_path.dart.

**Review by gjwgit — COMMENTED** [review]
I had to update to get it to work. Was this tested at all? 

Also non-conforming to style requirements.

Cleaned up coding.

Running `make qtest.tmp` to test before merge.

**Review by gjwgit — APPROVED** [review]

**Comment by yixiangyin** [comment]
> I had to update to get it to work. Was this tested at all?
> 
> Also non-conforming to style requirements.
> 
> Cleaned up coding.
> 
> Running `make qtest.tmp` to test before merge.

@gjwgit This is a **draft** PR only. I haven't made it actual PR because it's not ready to be merged yet.

**Pull request merged**